In-house patch

Fixes problem /w autoreconf which can not find m4 macro library.

Can be offered to community but this is HP who is "the community".

--- a/configure.in	2017-11-03 10:09:33.707678904 +0000
+++ b/configure.in	2017-11-03 10:10:33.386649869 +0000
@@ -25,6 +25,8 @@
 #   103 = no jpeg-devel
 #   104 = no libdl
 
+AC_CONFIG_MACRO_DIRS([m4])
+
 #AC_PREREQ(2.59)
 AC_INIT([HP Linux Imaging and Printing], [3.17.10], [3.17.10], [hplip])
 #AM_INIT_AUTOMAKE([1.9 foreign])
